packages/bot-web-ui/src/components/route-prompt-dialog/route-prompt-dialog.jsx

```diff
@@ -4,7 +4,7 @@ import { Dialog } from '@deriv/components';
 import { connect } from 'Stores/connect';
 
 const RoutePromptDialog = ({ continueRoute, is_confirmed, last_location, should_show, onCancel, onConfirm }) => {
-    React.useEffect(continueRoute, [is_confirmed, last_location]);
+    React.useEffect(continueRoute, [is_confirmed, last_location, continueRoute]);
```

Comment:
`continueRoute` is redundant

Reply:
React Hook useEffect has a missing dependency: 'continueRoute'. Either include it or remove the dependency array. (this is the warning that we are receiving).
This is the reason that I've included continueRoute as dependency. The other option is to remove the dependency array which is not recommended.

Reply:
Oh I see,
I was thinking that continueRoute is in dependency array!
that's fine.